erdpy/cli.py

```diff
@@ -188,8 +188,11 @@ def setup_parser_validators(subparsers):
     stake_prepare_parser_and_send = subparsers.add_parser("stake")
     stake_prepare_parser_and_send.add_argument("--pem", required=True)
     stake_prepare_parser_and_send.add_argument("--value", default="0")
+    stake_prepare_parser_and_send.add_argument("--estimate-gas", action="store_true", default=False)
     stake_prepare_parser_and_send.add_argument("--gas-price", default=config.DEFAULT_GASPRICE)
-    stake_prepare_parser_and_send.add_argument("--gas-limit", default=config.DEFAULT_GASLIMIT)
+    stake_prepare_parser_and_send.add_argument("--gas-limit",
+                                               required=not("--estimate-gas" in sys.argv),
+                                               default=config.DEFAULT_GASLIMIT)
```

Comment:
Please remove `default=config.DEFAULT_GASLIMIT` (and below). Not needed, correct?

Reply:
done.